build.sbt

```diff
@@ -152,6 +152,7 @@ lazy val jvmProjects: Seq[ProjectReference] = communityBuildProjects ++ Seq[Proj
   fluentd,
   airspecLight,
   finagle,
+  okhttp,
```

Comment:
airframe-okhttp was not included in airframe 20.3.2 release. I'll release 20.3.3 after merging this PR